podcasts/Ratings/StarRatingView.swift

```diff
@@ -29,7 +29,11 @@ struct StarRatingView: View {
         }.onTapGesture {
             viewModel.didTapRating()
         }
-        .sheet(isPresented: $viewModel.presentingGiveRatings) { RatePodcastView(viewModel: RatePodcastViewModel(presented: $viewModel.presentingGiveRatings)) }
+        .sheet(isPresented: $viewModel.presentingGiveRatings) {
```

Comment:
I saw the "Attempt to present ... which is already presenting" error a couple times in the console while testing. I think what is happening is the tableview is reloading and another rating view is created and attempts to present again. 

wdyt about instead of showing the sheet here, we can add a method to the `PodcastActionsDelegate` and on tap have the `PodcastViewController` display the sheet?

Reply:
Fixed in https://github.com/Automattic/pocket-casts-ios/pull/1185